### PR TITLE
first version of issue templates

### PR DESCRIPTION
**Pull request by timcappalli**
#1777

### CODE REVIEW

**Review by MasterKale — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/editorial-change--.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: "Editorial Change"
+about: Proposal for an editorial change to the WebAuthn specification
+labels: ["type:editorial"]
+---
+
+## Proposed Change
+
+Describe your proposed change
```
Comment (wseltzer) [inline .github/ISSUE_TEMPLATE/editorial-change--.md:9]: Possibly hint "an editorial change is one that doesn't affect the behavior of the spec. If your proposed change would affect behavior, even if minor, please file as technical."

Comment (wseltzer) [reply .github/ISSUE_TEMPLATE/editorial-change--.md:9]: and "If you have suggested text, please file a corresponding Pull Request."

**Review by wseltzer — APPROVED** [review]
LGTM with or without suggested changes

File: .github/ISSUE_TEMPLATE/editorial-change--.md (hunk 1 above)
Comment (timcappalli) [reply .github/ISSUE_TEMPLATE/editorial-change--.md]: Done. Thank you!

**Pull request merged**